### ISSUE TITLE
[feature request] add multiple selection to edit lists

### ISSUE THREAD

**Issue by illustribe** [TeamNewPipe/NewPipe#1956]
Hello, it would be wonderful if more than one item in the playlists com be selected to be edited it deleted, like in usual music players. Having to remove one by one is such a time consuming task

 Thank you!

- [x ] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x ] I checked if the issue/feature exists in the latest version.
- [x ] I did use the [incredible bugreport to markdown converter](https://teamnewpipe.github.io/CrashReportToMarkdown/) to paste bug reports.

**Comment by keks** [comment]
Yeah this would be awesome. What I would currently like to do is to select multiple videos in channel and create a playlist from them. Adding multiple selection would probably get me there.
